flameshot: add examples and fix more info link (#6377)
tldr-pages/tldr: pages/linux/flameshot.md

> Screenshot utility with a GUI.
> Supports basic image editing, such as text, shapes, colors, and imgur.
> More information: <https://flameshot.org>.

> - Launch flameshot launcher window:

Reviewer: ```suggestion
- Launch flameshot with an interactive mode:
```

Although technical terms have their place, the description is best when the user can gather some information from it.

`flameshot launcher`

> - The GUI mode to take a screenshot by clicking and dragging and annotating interactively:

Reviewer: ```suggestion
- Launch flameshot and immediately start interactively annotating parts of the screen to screenshot.
```

> - Launch flameshot and immediately start interactively annotating parts of the screen to screenshot.

Reviewer 2: ```suggestion
- Launch flameshot and immediately start interactively annotating parts of the screen to screenshot:
```

`flameshot gui`

- Take a full screenshot (all monitors):

`flameshot full`

> - Take screenshot from monitor 1:

Reviewer 3: Just the teensiest of suggestions
```suggestion
- Take a screenshot from monitor 1:
```

> `flameshot screen --number 1`

Reviewer: ```suggestion
`flameshot screen --number {{1}}`
```

Our syntax for variables

- Set the save path to write screenshots to:

`flameshot full --path {{path/to/directory}}`

- Delay the screenshot for N milliseconds and output to clipboard:

`flameshot full --delay {{2000}} --clipboard`

> - Export the screenshot to standard-output:

Reviewer: ```suggestion
- Take a screenshot and export it to standard-output:
```
